staging/src/k8s.io/apiserver/pkg/cel/library/authz.go

```diff
@@ -178,6 +178,11 @@ func Authz() cel.EnvOption {
 	return cel.Lib(authzLib)
 }
 
+// AuthzLibrary returns the cel Library declaration.
+func AuthzLibrary() SupportedLibrary {
```

Comment:
a sketch of how we would use this (maybe by going ahead and getting the versioned wiring in place for "version 0") would make it clearer how this would work

Reply:
I'd like to back this change out and work on library changes in a separate PR.